internal/driver/driver.go

```diff
@@ -187,80 +184,23 @@ func (d *Driver) Initialize(lc logger.LoggingClient, asyncCh chan<- *dsModels.As
 	return nil
 }
 
-func (d *Driver) watchForConfigChanges() error {
-	sdkFlags := flags.New()
-	sdkFlags.Parse(os.Args[1:])
-	cpUrl, err := url.Parse(sdkFlags.ConfigProviderUrl())
-	if err != nil {
-		return err
+func (d *Driver) updateWritableConfig(rawWritableConfig interface{}) {
+	updated, ok := rawWritableConfig.(*CustomConfig)
+	if !ok {
+		d.lc.Error("unable to update writable custom config: Can not cast raw config to type 'CustomConfig'")
+		return
 	}
 
-	cpPort := 8500
-	port := cpUrl.Port()
-	if port != "" {
-		cpPort, err = strconv.Atoi(port)
-		if err != nil {
-			cpPort = 8500
-		}
-	}
+	d.configMu.Lock()
+	oldSubnets := d.config.AppCustom.DiscoverySubnets
+	oldScanPort := d.config.AppCustom.ScanPort
+	d.config.AppCustom = *updated
+	d.configMu.Unlock()
 
-	configClient, err := configuration.NewConfigurationClient(types.ServiceConfig{
-		Host:     cpUrl.Hostname(),
-		Port:     cpPort,
-		BasePath: baseConsulPath,
-		Type:     strings.Split(cpUrl.Scheme, ".")[0],
-	})
-	if err != nil {
-		return err
+	if updated.DiscoverySubnets != oldSubnets || updated.ScanPort != oldScanPort {
+		d.lc.Info("discover configuration has changed!")
```

Comment:
Yes, this should be Info. The details should remain Debug.

Reply:
I think I was more trying to get at the fact we might want to add a note that it will be auto-triggered. I'm not sold on the wording, but this was my thoughts:
```suggestion
		d.lc.Info("Discover configuration has changed! Discovery will be triggered momentarily.")
```

Reply:
I added the note you suggested